services_app/src/main/res/values/strings.xml

```diff
@@ -290,6 +290,8 @@
 
     <string name="enter_new_user">Enter new username:</string>
     <string name="enter_new_password">Enter new password:</string>
+    <string name="enter_server_password">Server Password:</string>
+    <string name="new_server_password">Enter New Password</string>
```

Comment:
This seems like a duplicate between "enter_new_password" and "new_server_password"

Reply:
Oops... yes that is right